### DISCUSSION TITLE
404 for static sites

### DISCUSSION THREAD

**Discussion by vesparny** [vercel/next.js#14918] [Help]
Hi, I see next is able to serve 404 responses also for static site.
How is that possible considering there is no server running?

Thanks

**Comment by cristiand391** [comment]
It depends on which platform you use to deploy your app.

If you deploy it on Vercel, static pages and assets are server from the Vercel Smart CDN.
If you choose another platform, a Node.js server is used to serve both SSG and SSR pages.

You can learn more in the [Next.js Deployment docs](https://nextjs.org/docs/deployment).
